tests/rkt_exit_test.go

```diff
@@ -71,13 +71,16 @@ func TestExitCodeWithSeveralApps(t *testing.T) {
 	}
 
 	t.Logf("Waiting pod termination\n")
-	waitOrFail(t, child, true)
+	// Currently we have systemd v222 in the coreos flavor which doesn't
+	// include the exit status propagation code.
+	// TODO(iaguis): we should expect 5 as the exit status when we update to v229
+	waitOrFail(t, child, 0)
```

Comment:
I'd like if tests/aws.sh could test the `host` flavor on all versions of Fedora... this would require something like...?
```
if systemdVersion() > 227 {
  waitOrFail(t, child, 5)
} else {
  waitOrFail(t, child, 0)
}
```